index.html

```diff
@@ -144,7 +144,7 @@ <h2>A project for first-time contributions</h2>
 
 
 
-         <!-- ________ Seo-Asif card START ________  -->
+         <!-- ________ Seo-Asif card START ________  --->
```

Comment:
All you'd have to do is move it above Jeremiahs card and I will merge it. Thanks!